src/features/profile/ui/user-comment-list.tsx

```
@@ -0,0 +1,111 @@
+          {/* We strictly don't know text height without Ref, but CSS masking is a good heuristic */}
+          {!isExpanded && (comment.content.length > 300 || comment.content.includes('<img')) && (
+            <div className="from-background/10 to-background absolute bottom-0 left-0 right-0 h-12 bg-gradient-to-b" />
+          )}
+        </div>
+
+        {(comment.content.length > 300 || comment.content.includes('<img')) && (
+          <button
+            onClick={() => setIsExpanded(!isExpanded)}
+            className="text-primary hover:text-primary/80 mb-3 flex items-center text-sm font-medium"
+          >
+            {isExpanded ? (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)